microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -20,118 +20,188 @@ keywords: microsoft edge, web development, f12 tools, devtools
-    
-    :::image type="complex" source="../media/issues-tab-after-refresh.msft.png" alt-text="Issues tool in the DevTools Drawer with three cookie issues" lightbox="../media/issues-tab-after-refresh.msft.png":::
-       **Issues** tool in the DevTools Drawer with three cookie issues  
-    :::image-end:::  
-    
-## View items in the Issues tool  
+The **Issues** tool supports provides feedback in the following categories:
+*  **Accessibility**
+*  **Compatibility** (cross-browser compatibility)
+*  **Other**
+*  **Performance**
+*  **Progressive Web Apps**
+*  **Security**
 
-The **Issues** tool in the DevTools Drawer presents warnings in a structured, aggregated, and actionable way.  
+The feedback in the **Issues** tool is provided by several sources, including the Chromium platform, Deque axe, the MDN Browser Compatibility Report, and webhint.  For information about these sources of feedback that populate the Issues tool, navigate to:
+*  [axe Tools Overview][DequeAxe]
+*  [Compat2021: Eliminating five top compatibility pain points on the web][MDNCompat]
+*  [webhint][webhintIo]
+
+### The Issues counter and Issues tool are dynamically updated
+
+When you open a page, sometimes the Issues counter \(![Issues counter](../media/issues-counter-icon.msft.png)\) doesn't immediately appear in DevTools, but is displayed after one or two seconds.
+
+When you refresh a page, the Issues counter sometimes increases the count of issues.  The count is updated and the Issues tool lists more issues, as analysis results come in from several sources.
+
+
+## Open the Issues tool
+
+The following steps walk through opening the Issues tool, using a couple of demo pages.
+*  The accessibility-testing demo page has issues in the **Accessibility**, **Compatibility**, **Performance**, and **Security** categories.
+*  The SameSite cookies demo page has issues in the **Other** and **Security** categories.
+
+To open the issues tool:
+
+1.  Navigate to a webpage that contains issues to fix.  For example, open the [accessibility-testing demo page](https://microsoftedge.github.io/DevToolsSamples/a11y-testing/page-with-errors.html) in a new tab or window.
+
+    :::image type="complex" source="../media/issues-accessibility-demo-page-initial.msft.png" alt-text="Accessibility-testing demo page, which has some issues" lightbox="../media/issues-accessibility-demo-page-initial.msft.png":::
+       Accessibility-testing demo page, which has some issues
+    :::image-end:::
+
+    Or, open the [SameSite cookies demo page][GlitchSamesiteSandbox] in a new tab or window.  This page initially brings up a warning page, which reads `This site has been reported as unsafe`, `Hosted by samesite-sandbox.glitch.me`.  Select **More information** to expand the section, and then select **Continue to the unsafe site (not recommended)**.  We won't enter information in the demo page.
```

Comment:
I think it's more confusing than helpful to have two different demo pages, especially since the examples are interspersed. The accessibility demo shows the majority of issue types, so I'd rather keep it to just that one example.